### ISSUE TITLE
Support User Feedback feature of Sentry

### ISSUE THREAD

**Issue by laralove143** [getsentry/sentry-rust#672]
The message says to submit user feedback through the web API as per [these docs](https://docs.sentry.io/api/projects/submit-user-feedback/). It would be better DX to have a method to do that, rather than having developers write their own API wrapper.

**Comment by zmitchell** [comment]
I'm not sure what message they're referring to either, but I think they're just asking for the User Feedback API to be supported in the Rust SDK. The Web API specifically says this:

> This endpoint is DEPRECATED. We document it here for older SDKs and users who are still migrating to the [User Feedback Widget](https://docs.sentry.io/product/user-feedback/#user-feedback-widget) or [API](https://docs.sentry.io/platforms/javascript/user-feedback/#user-feedback-api)(multi-platform). If you are a new user, do not use this endpoint - unless you don't have a JS frontend, and your platform's SDK does not offer a feedback API.

The Rust SDK does not offer a feedback API, hence the request. Do you intend to support that feature in the Rust SDK or not?

**Comment by laralove143** [comment]
Yes, you could replace my original link with [the new API](https://docs.sentry.io/platforms/javascript/user-feedback/#user-feedback-api), which as far as I’m aware isn’t supported in the Rust crate.

**Comment by zmitchell** [comment]
I took a stab at implementing the feature myself here: https://github.com/getsentry/sentry-rust/issues/743

It's not working yet, and I don't know why. I'm happy to upstream the change if a maintainer can tell me what I'm missing.
